src/triage/exceptions.py

```diff
@@ -0,0 +1,13 @@
+
+__all__ = ['BaselineFeatureNotInMatrix']
+
+
+class BaselineFeatureNotInMatrix(Exception):
```

Comment:
It looks like triage catches this exception, rather than emitting it to the caller; and, the exception is only used within catwalk. So, is it really a triage exception? It seems more like a catwalk-only exception.